packages/repository-tests/src/types.repository-tests.ts

```diff
@@ -38,6 +39,15 @@ export interface CrudConnectorFeatures {
    * Default: `true`
    */
   freeFormProperties: boolean;
+
+  /**
+   * Does the connector support using transactions for performing CRUD
+   * operations atomically and being able to commit or rollback the changes?
+   * SQL databases usually support transactions
+   *
+   * Default: `false`
+   */
+  supportsTransactions: boolean;
```

Comment:
`supportsTransaction`?

Reply:
I personally prefer `supportsTransactions` (plural form).

Singular form reads weird to me - is it "supports _a_ transaction" or "supports _the_ transaction"? Which particular transaction do we have in mind?

Reply:
I've left it as `supportsTransactions` as in the transaction feature set. I don't mind changing it if we feel strongly about `supportsTransaction`.

Reply:
I'm good with `supportsTransactions`.

Reply:
On the second thought, I would use just `transactions`. The structure is called `CrudFeatures`, it describes what the connector and/or the repository supports - no need to repeat `support` in the flag name again.

Not a big deal though.

Reply:
Okay leaving it as is for now :).